docker/README.md

```diff
@@ -1,55 +1,36 @@
 # Welcome to the new Docker based Open Library development environment!
 
-These Dockerfiles replace the previous Vagrant based development environment.
-
 ## Setup/Teardown Commands
 
 ### For Windows Users Only
 
-**Note:**
-If you get permission issues while executing these commands please run git bash shell as an Administrator.
-
-1. Set the line endings to type `input` by executing the following command:
-
-   ```bash
-   # Configure Git on Windows to properly handle line endings so Git will convert CRLF to LF on commit
-   git config --global core.autocrlf input
-   ```
-
-2. Check whether `symlinks` are enabled or not. `openlibrary` makes use of symlinks which, by default, git on Windows checks out as plain text files. You can check that by executing the following command:
-
-   ```bash
-   git config core.symlinks
-   ```
+**Note:** If you get permission issues while executing these commands please run git bash shell as an Administrator.
 
-3. If `symlinks` are enabled then go to the next step. If `symlinks` are not enabled then enable it by executing the following command:
-
-   ```bash
-   git config core.symlinks true
-   ```
-
-4. Then hard reset the repository so that git will create proper symlinks by executing the following command:
+```bash
+# Configure Git to convert CRLF to LF line endings on commit
+git config --global core.autocrlf input
 
-   ```bash
-   git reset --hard HEAD
-   ```
+# Enable Symlinks
+git config core.symlinks true
 
-5. Continue setup as shown below.
+# Reset the repo (removes any changes you've made to files!)
+git reset --hard HEAD
+```
 
 ### For All Users
 All commands are from the project root directory:
 
 ```bash
 # build images
-docker build -t olbase:latest -f docker/Dockerfile.olbase .
-docker build -t oldev:latest -f docker/Dockerfile.oldev .
-docker build -t olsolr:latest -f docker/Dockerfile.olsolr .
+docker build -t olbase:latest -f docker/Dockerfile.olbase . # 30+ min (Win10Home/Dec 2018)
```

Comment:
Will this work without first doing a `docker-machine start`?

Reply:
I think it depends on your setup. What OS are you using? Have you had issues with this when you were setting things up?

Reply:
Yeah, I have to do it on Windows as well. Adding a section for it.

Reply:
Ok, according to the docs, that only needs to be run if you're using Docker Toolbox as opposed to regular docker. Updated accordingly.